init-pants/README.md

```diff
@@ -0,0 +1,87 @@
+
+`gha-cache-key`: This is used to create the GHA cache keys for pants' `lmdb_store`
+and `named_caches`. When pulling the pants files from the GHA cache,
+this can be used to include relevant metadata (such as the python version your app is using),
+or to bust the cache, discarding old versions of the cached pants metadata.
+
+`named-caches-hash`: This is used to create the GHA cache key for pants' `named_caches`.
+Pants keeps pip and pex caches in the named caches, so they need to be invalidated
+when transitive dependencies change. The `named-caches-hash` should use the
+`hashFiles()` function to create a hash of the relevant files. We recommend using
+lockfiles for this, because named caches are primarily used today for the results of
+resolving lockfiles. If you aren't using lockfiles, you can also hash
+`requirements.txt` and any `BUILD` files that define other dependencies.
+For example: `${{ hashFiles('lockfiles/*.json') }}`
+
+### Optional input arguments
+
+`pants-python-version`: Which version of python to install, specifically for pants. Defaults to `'3.9'`.
+
+`base-branch`: This action calculates the merge base for pull requests from this branch.
+Looking up the merge commit allows us to use the cache from the latest commit on the base branch.
+
+`pants-ci-config`: The value for the `PANTS_CONFIG_FILES` environment var.
+Set to empty to skip adding it to the environment for the rest of the workflow.
+Defaults to `pants.ci.toml`.
+For more about this var and the file naming convention, see:
+https://www.pantsbuild.org/docs/using-pants-in-ci#configuring-pants-for-ci-pantscitoml-optional
+
+`cache-lmdb-store`: Pass the string `'true'` to enable caching pamts' lmdb store in
+a GHA cache. By default, this action does NOT cache the `lmdb_store`.
+This is a very coarse cache that can grow unbounded. So, it is very likely to hit
+GitHub's 10GB per repo max for action caches. If you enable this, you need another
+process or workflow to manage discarding older GHA caches, or minimizing the cache size
+as described in the [docs](https://www.pantsbuild.org/docs/using-pants-in-ci).
+Use the default if using [remote caching](https://www.pantsbuild.org/docs/remote-caching).
+
+## Secrets
+
+This action does not use any secrets at this point. It might need some once it supports projects that use remote caching.
+
+## Environment variables
+
+This action sets the `PANTS_CONFIG_FILES` environment var using the value of the `pants-ci-config` input parameter.
+The environment variable should be available for the remainder of the workflow that uses this action.
+
+## Usage Example
+
+An example of how to use this action in a workflow:
+
+```yaml
+      - name: Initialize Pants
+        uses: pantsbuild/actions/init-pants@main
+        with:
+          # cache0 makes it easy to bust the cache if needed
+          gha-cache-key: cache0-py${{ matrix.python_version }}
+	  named-caches-hash: ${{ hashFiles('lockfiles/*.json', '**/something-else.lock') }}
```

Comment:
Is this indicating a tab rather than spaces?

Reply:
Yes. Oops. I'm doing this on my phone, so I missed some of my typos.